src/roguelike/__main__.py

```diff
@@ -0,0 +1,19 @@
+"""
+Contains game main loop
+"""
+
+import os
+
+from roguelike.game_engine.game_manager.game_constructor.game_loader import GameLoader
+from roguelike.game_engine.game_manager.game_processor.game_loop import GameLoop
+from roguelike.ui.keyboard_interpreter import KeyboardInterpreter
+
+
+if __name__ == '__main__':
+    file_path = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'assets/maps/forest.json')
```

Comment:
let's introduce variable for this magic constant

Reply:
fixed this, thanks